src/app/search/search-tool-table/search-tool-table.component.ts

```diff
@@ -14,8 +15,8 @@ import { SearchQuery } from '../state/search.query';
 })
 export class SearchToolTableComponent extends SearchEntryTable implements OnInit {
   public dataSource: MatTableDataSource<DockstoreTool>;
-  constructor(private dockstoreService: DockstoreService, protected dateService: DateService, private searchQuery: SearchQuery) {
-    super(dateService);
+  constructor(private dockstoreService: DockstoreService, protected dateService: DateService, protected searchQuery: SearchQuery, protected searchService: SearchService) {
```

Comment:
You shouldn't need `protected' or any modifier since you are just passing them to super constructor.